QgisModelBaker/libqgsprojectgen/dataobjects/fields.py

```diff
@@ -32,6 +33,7 @@ def __init__(self, name):
         self.widget = None
         self.widget_config = dict()
         self.default_value_expression = None
+        self.mandatory = None
```

Comment:
Can we call this `not_null` to be consistent in terminology?

Reply:
Done!